### ISSUE TITLE
Skip as button started loading even after the ad playback is paused

### ISSUE THREAD

**Issue by vinoddirishala** [flipkart-incubator/madman-android#23]
The skip ad button is completing it's offset even the ad playback is paused. Are there any callbacks to handle this ?

**Comment by vinoddirishala** [comment]
![skipAD madman](https://user-images.githubusercontent.com/22494324/100366891-7170ae80-3027-11eb-9652-5a2ed0bb3f6f.jpeg)

**Comment by anirudhramanan** [comment]
No, the DefaultAdRenderer runs a separate count down timer for skip ads.

This is part of our backlog, will be picked up soon. The ideal case is where the AdManager constantly gives state / progress callbacks to the AdRenderer, and the AdRenderer handles such cases.

**Comment by anirudhramanan** [comment]
The fix has been merged to develop. You can use jitpack for now to create a snapshot from develop to use
